package.json

```diff
@@ -40,15 +41,15 @@
     "iced-coffee-script": "^108.0.11",
     "marked": "^0.3.7",
     "marked-terminal": "^2.0.0",
-    "moment": "~2.18.1",
+    "moment": "~2.21.0",
```

Comment:
Can you change the `~` to `^` for moment. I had added the `~` few months ago because moment broke something in the 2.19.0 version. It seems they fixed it in 2.20.0. Hence we should be good to move to `^2.21.0` which means `2.x.x`

Reply:
Done.